baresipy/__init__.py

```
@@ -171,17 +186,25 @@ def quit(self):
             self.baresip.sendline("/quit")
-        self.running = False
-        self.current_call = None
-        self._call_status = None
-        self.abort = True
-        self.baresip.close()
-        self.baresip.kill(signal.SIGKILL)
+            self.baresip.close()
+            self.baresip.kill(signal.SIGKILL)
+            stopped_now = True
+        
+        if(stopped_now):
```

Flake8 (7.2.0)

[error] missing whitespace after keyword

(E275)